packages/react-menu/.storybook/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true
+  },
+  "exclude": ["../**/*.spec.ts", "../**/*.spec.js", "../**/*.spec.tsx", "../**/*.spec.jsx"],
```

I think that we still don't use `spec` as an extension yet ? Might also want to add `e2e`

```suggestion
  "exclude": ["../**/*.test.ts", "../**/*.test.js", "../**/*.test.tsx", "../**/*.test.jsx",  "../**/*.e2e.tsx"],
```

---

ah right 🤦‍♀️. I'm gonna add whole `e2e/` instead of `.e2e.tsx`. As discussed previously those should be strictly separated.

---

oh actually that's not needed as `/e2e` is sibling of `/src`

---

ah lucky it just all falls into place 😁